### DISCUSSION TITLE
Borderless Window Mod

### DISCUSSION THREAD

**Discussion by 8-Pit** [geode-sdk/sdk#397] [Mod Ideas]
Borderless Window/Windowed Fullscreen mod would be a very good mod. It's much better than standard fullscreen, especially with multiple monitors, since the game doesn't stay opened as soon as you do literally anything on a secondary monitor.

**Comment by TechStudent10** [comment]
Well we can mark this as "RobTop added it already"

**Reply by 8-Pit** [reply]
I guess so lol

**Comment by Cuteistfox** [comment]
Geode is a modloader, they dont make mods, also this is closed by robtop
